pageserver/src/task_mgr.rs

```diff
@@ -453,7 +451,7 @@ async fn task_finish(
     }
 
     if shutdown_process {
-        shutdown_pageserver(None, 1).await;
+        std::process::exit(1);
     }
```

Comment:
I suppose this fixes https://github.com/neondatabase/neon/issues/3402 ?

Reply:
Good find -- I had no idea that ticket existed.  Yes, I've just closed it.